Source/cmake/OptionsJSCOnly.cmake

```diff
@@ -108,3 +108,7 @@ if (USE_LIBBACKTRACE)
         message(FATAL_ERROR "libbacktrace is required for USE_LIBBACKTRACE")
     endif ()
 endif ()
+
+if (ENABLE_DEVELOPER_MODE)
+    WEBKIT_PREPEND_GLOBAL_COMPILER_FLAGS(-DENABLE_OFFLINE_ASM_ALT_ENTRY=1)
+endif ()
```

Comment:
It is JSCOnly port's flag. I think probably you  need to put it to `Source/JavaScriptCore/CMakeLists.txt`